test/unit/context.test.ts

```diff
@@ -176,7 +156,7 @@ describe("Context", () => {
              * to update them
              */
             if (process.env.UPDATE_CONTEXT_FIXTURE) {
-              const snapshotPath = "../fixtures/context"
+              const snapshotPath = '../fixtures/context'
               writeFileSync(
                 join(__dirname, snapshotPath, `fetchContext${f.index}.json`),
```

Comment:
This should be updated to generate the required `.ts` files rather than `.json` files

Reply:
I didn't change this test other than the auto formatter so this is a good case for a future PR.